spec/rubocop/cop/rspec/pending_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe RuboCop::Cop::RSpec::Pending do
+  subject(:cop) { described_class.new }
+
+  # rubocop:disable RSpec/ExampleLength
+  it 'flags all pending specs' do
+    expect_offense(<<-RUBY)
+      it 'test' do; skip; end
+                    ^^^^ Pending spec found.
```

Comment:
I think this example would actually work better as separate examples (just as RSpec/ExampleLength suggests). If the code didn’t work for e.g. `xfeature`, I imagine the RSpec output would be pretty hard to read. Much simpler if there were a spec example like
```rb
it 'flags xfeature' do
  expect_offense <<-RUBY
    xfeature  'test' do
    ^^^^^^^^^^^^^^^^ Pending spec found.
    end
  RUBY
end
```